transport/src/test/java/io/netty/channel/CoalescingBufferQueueTest.java

```diff
@@ -67,14 +65,7 @@ public void operationComplete(ChannelFuture future) throws Exception {
 
         cat = Unpooled.wrappedBuffer("cat".getBytes(Charset.defaultCharset()));
         mouse = Unpooled.wrappedBuffer("mouse".getBytes(Charset.defaultCharset()));
-        empty = Unpooled.buffer(0, 1);
-    }
 
-    @After
-    public void tearDown() {
-        ReferenceCountUtil.safeRelease(cat);
-        ReferenceCountUtil.safeRelease(mouse);
```

Comment:
Yes, sorry I thought I was releasing them in all the tests but that is not the case for the 'empty' tests.